docs: Change important alert of ngFor

bampakoa opened the pull request:
Update the important alert of `ngFor` in the **Displaying Data** guide so that it has a unique format with that of `ngIf`.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

bampakoa committed:
docs: Change important alert of ngFor

Update the important alert of ngFor so that it has a unique format with that of ngIf

gkalpak commented on aio/content/guide/displaying-data.md:
```diff
@@ -153,7 +153,7 @@ It marks that `<li>` element (and its children) as the "repeater template":
 
 Don't forget the leading asterisk (\*) in `*ngFor`. It is an essential part of the syntax.
-For more information, see the [Template Syntax](guide/template-syntax#ngFor) page.
+Read more about `ngFor` and `*` in the [ngFor section](guide/template-syntax#ngfor) of the [Template Syntax](guide/template-syntax) page.
```
Good idea :+1: However, in contrast to the `ngIf` section, the `ngFor` section does not contain info about the `*`.
Maybe we could add an alert similar to [this one][1]?

[1]: https://github.com/angular/angular/blame/528e25a81a90c828365a2df08328f55be01e421b/aio/content/guide/template-syntax.md#L1528-L1534

bampakoa replied:
@gkalpak you are right 👌 You mean to add the alert in the `ngFor` section of the template syntax, yes?

gkalpak replied:
Yes :+1:

gkalpak reviewed (changes requested):
Thx, @bampakoa!

bampakoa committed:
docs: Add asterisk info in template syntax guide

Add helpful alert for asterisk syntax in the `ngFor` section of template syntax guide

gkalpak reviewed (approved):
Thx, @bampakoa - LGTM 👌

gkalpak commented:
**merge-assistance:** Global approval for this docs-only change.
